README.md

```diff
@@ -1,55 +1,61 @@
-# FlowCrypt: Encrypt Gmail with PGP
+# FlowCrypt: Encrypt Gmail with OpenPGP
 
 ## Users
 
-Get [FlowCrypt](https://flowcrypt.com/) browser extension at: https://flowcrypt.com/download
+Get the [FlowCrypt](https://flowcrypt.com/) browser extension from the [FlowCrypt downloads](https://flowcrypt.com/download) page.
 
 ## Developers
 
 Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-browser.svg?key=d575b4ff-e35d-4217-9152-27cea9e72c19)
 
 Live tests: [![Live Tests](https://semaphoreci.com/api/v1/flowcrypt/flowcrypt-browser/branches/master/badge.svg)](https://semaphoreci.com/flowcrypt/flowcrypt-browser)
 
-This project is written in TypeScript. Browsers only understand JavaScript, so the project needs to be transpiled. You need to build the project the first time you download it, and build it after each change you make to see the result in the browser. To get started, please follow the instructions below:
+This project has been written in TypeScript. However, since browsers only understand JavaScript, the project needs to be transpiled to it. You need to build the project the first time you download/clone it, and then build it again after each change you make to see the result in the browser. To get started, please follow the instructions below:
```

Comment:
I think `is written` is more correct, as it means `project writing continues`, while `has been written` sounds like `project writing is finished`

Reply:
What if we use "We develop the project..." in that case?

Reply:
Sounds good too 👍

Reply:
\+